nfiles.lisp

```diff
@@ -435,7 +469,7 @@ See `*gpg-default-recipient*'."
   (declare (ignore destination))
   nil)
 
-(defun backup (path)
+(defun back-up (path)
```

Comment:
I'd rather rename it with a percent sign, so that it's clear that the function is internal to the restart. Two different spellings are quite confusing.

Reply:
It's actually proper English.  "Backup" is a noun, "back up" is a verb.  Perfect for a function.

Reply:
But then `backup`, as a restart, is an action/verb too.

In any way, the spelling is inconsistent, so I'd rather have one spelling (we can settle at `back-up`) and a percent sign in front of the internal function, if possible. Percent sign is better than a linguistic meaning :)

Reply:
But the restart is already exported, so we can't change it.

Inconsistencies for internal functions are not a big deal.
`%backup` is not a very good name for a function.

Reply:
Let it be then ╮(─▽─)╭